images/scripts/rhel-8-3.bootstrap

```diff
@@ -3,4 +3,4 @@
 set -ex
 
 BASE=$(dirname $0)
-$BASE/virt-install-fedora "$1" x86_64 "http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.3/compose/BaseOS/x86_64/os/"
+$BASE/virt-install-fedora "$1" x86_64 "http://download.devel.redhat.com/rhel-8/nightly/updates/RHEL-8/latest-RHEL-8.3.1/compose/BaseOS/x86_64/os/"
```

Comment:
The inclusion of `updates` is suspicious here.  I wouldn't be surprised if one can not bootstrap from any `update` repository... but then again `/rhel-8/nightly/updates/RHEL-8/latest-RHEL-8.0.0/compose/BaseOS/x86_64/os/` does have isolinux/... I will never understand how RHEL is made...

Reply:
See #1425 for a try with just latest-RHEL-8.3.1.

Reply:
That's a good point -- let's keep the release repo here, and only apply updates in the setup script (as it already does).

Reply:
Why would a nightly build already have updates?

Reply:
Ah right, I somehow thought .1 was already out, but of course the point of all this is exactly to test the *pre*releases.